main.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"encoding/csv"
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/gorilla/mux"
+)
+
+// Movie struct
+type Movie struct {
+	ID       int    `json:"ID"`
+	Title    string `json:"Title"`
+	Director string `json:"Director"`
+}
+
+// Movies list
+var Movies []Movie
```

Comment:
For starting it's okay to have everything inside the main, but better start working on clean architecture principles. Please,give it another look to clean architecture document.

According to clean architecture, this should be moved into a `model` package.